go/roothash/api/commitment/pool.go

```diff
@@ -205,8 +210,8 @@ func (p *Pool) addOpenExecutorCommitment(
 	}
 
 	// Verify that the txn scheduler signature for current commitment is valid.
-	if err := sv.VerifyCommitteeSignatures(scheduler.KindComputeTxnScheduler, []signature.Signature{body.TxnSchedSig}); err != nil {
-		logger.Debug("executor commitment has bad transaction scheduler signers",
+	if err := sv.VerifyTxnSchedulerSignature(body.TxnSchedSig, height); err != nil {
```

Comment:
The pool should probably remember the height at which the first valid commitment was received for the current round and then use that for subsequent commitments (until reset at end of round)?